packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -7,26 +7,31 @@ import UnitRow from '../UnitRow';
 import UnitRowWithAvatar from '../UnitRowWithAvatar';
 import Security from '../Security';
 import UnitRowSecondaryEmail from '../UnitRowSecondaryEmail';
-import { Account } from '../../models';
 
-export const Settings = ({ account }: { account: Account }) => {
-  const primaryEmail = account.emails.find((email) => email.isPrimary)!;
+import { useAccount } from '../../models';
+
+const {
+  emails,
+  displayName,
+  avatarUrl,
+  passwordCreated,
+  recoveryKey,
+} = useAccount();
```

Comment:
This should be in the `Settings` component function because `useAccount()` needs to be called after the app has initialized instead of when this file is first loaded.

Reply:
Just pushed a fix for this